doc/source/Tutorials/GettingStarted/index.md

```diff
@@ -1,407 +1,499 @@
+4. Create an Outputs node (_Scene_ > _Globals_ > _Outputs_). It will connect and become selected like the others.
 
-The viewer should update to show the new assignment.
+5. Create an InteractiveAppleseedRender node (_Appleseed_ > _InteractiveRender_).
 
-![Second shader assignment render](images/secondShaderAssignmentRender.png)
+6. Create a Catalogue node (_Image_ > _Utility_ > _Catalogue_). It does not need to be connected to anything. Instead, place it next to the InteractiveAppleseedRender node.
 
-Oops. Because the new ShaderAssignment is downstream from the first one, it has overridden the first assignment, and now _everything_ is chrome. We need a way of limiting the objects that the second ShaderAssignment is applied to - in Gaffer this is referred to as _filtering_ the ShaderAssignment, and it is done using special Filter nodes :
+    ![The render-related nodes](images/graphEditorRenderSettings.png "The render-related nodes")
 
-- Create a PathFilter node (_/Scene/Filters/PathFilter_). This chooses which objects to affect based on their names.
-- Use the NodeEditor to add `/group/GAFFERBOT/C_torso_GRP/C_head_GRP/C_head_CPT/L_ear001_REN` to the **Paths**. This is the
-  full name of the left ear of the robot, as you see it listed in the HierarchyView.
-- Connect the output of the PathFilter into the filter input on the right hand side of the ShaderAssignment1 node.
+Briefly, here is the function of each of these nodes:
+- StandardOptions node: Determines the camera, resolution, and blur settings of the scene.
+- AppleseedOptions node: Determines the settings of the Appleseed renderer.
+- Outputs node: Determines what kind of output render will be created.
+- InteractiveAppleseedRender node: An instance of Appleseed's interactive (responsive) renderer.
+- Catalogue node: Global list of images/renders that you can preview within Gaffer. Renders you generate will appear in this node's list.
 
-![Filter connection](images/filterConnection.png)
+In keeping with what we said earlier about nodes passing scenes to other nodes: with the exception of the Catalogue node, each of these render options nodes only apply to the scene delivered to them through their input plugs. If you had another unconnected scene running in parallel, none of these render settings would apply to it.
 
-The render should now update, with the chrome shader applied only to the ear.
+Although the scene contains a camera, you will need to point the StandardOptions node to it:
 
-Editing the filter
-------------------
+1. Select the StandardOptions node.
 
-We might like to apply the chrome shader to more objects, but it'll be a bit tedious to continue entering object names manually as
-we just did. First, let's use Gaffer's pattern matching to pick both ears :
+2. Specify the camera using the _Node Editor:_
+	1. Click the _Camera_ section to expand it.
+	2. Toggle the switch next to the Camera plug to enable it.
+	3. Type `/group/camera` into the plug's field.
```

Comment:
Can we have a Tip explaining the drag/drop behaviour from the _Viewer_ and _Hierarchy View_ ?